bin/send_email.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import sys
+import synapseclient
+
+
+def get_participant_id(syn, submission_id):
+    """
+    Retrieves the teamId of the participating team that made
+    the submission. If the submitter is an individual rather than
+    a team, the userId for the individual is retrieved.
+    """
+    # Retrieve a Submission object
+    submission = syn.getSubmission(submission_id, downloadFile=False)
+
+    # Get the teamId or userId of submitter
+    participant_id = submission.get("teamId") or submission.get("userId")
```

Comment:
Have you tested this for when there is a `teamId` in the `Submission` object? Does it notify everyone on the team?

Reply:
Hi @BWMac that's a good point - I just made a [submission](https://www.synapse.org/#!Synapse:syn53461697/tables/) to the DPE-testing project where I submitted on behalf of a test team. Please let me know if you don't receive the "Evaluation Success" e-mail (should look like the screenshot in the description).

Reply:
I did receive the email